client/src/components/layers/MarkerLayer.ts

```diff
@@ -44,7 +44,7 @@ export default class MarkerLayer extends BaseLayer<FormattedMarkerFeature> {
     return {
       ...baseStyle,
       fill: true,
-      fillColor: (data) => (data.feature === 'tail' ? 'orange' : 'blue'),
+      fillColor: (_point, _index, data) => (data.feature === 'tail' ? 'orange' : 'blue'),
```

Comment:
```
vue.runtime.esm.js?2b0e:1888 TypeError: Cannot read property 'feature' of undefined
    at webgl_pointFeature.fillColor (MarkerLayer.ts?56b5:47)
    at out (geo.lean.js?7edf:13608)
    at createGLPoints (geo.lean.js?7edf:72085)
    at webgl_pointFeature._build (geo.lean.js?7edf:72357)
    at webgl_pointFeature._update (geo.lean.js?7edf:72377)
    at webgl_pointFeature.draw (geo.lean.js?7edf:15178)
    at MarkerLayer.redraw (MarkerLayer.ts?56b5:53)
    at MarkerLayer.changeData (BaseLayer.ts?e8b1:85)
    at updateLayers (Layers.vue?5473:141)
    at setup (Layers.vue?5473:181)
```

Looks like you changed the arguments, but whatever actually calls this is passing the old arguments.  Not sure what's up.

Reply:
We have the BaseLayer.ts that we use to specify the interface for all stlyes.  I stopped thinking and assumed that `fillColor` was the same for all layers, but it's different for a Point Feature compared to a polygon feature.  The interface just needs to be updated to accept either a PolygonFeature fillColor style or a PointFeature fill style.

Reminds me also that I should make sure my test data has head/tail features in it.